dbt_project.yml

```diff
@@ -31,8 +31,15 @@ clean-targets:         # directories to be removed by `dbt clean`
 # In this example config, we tell dbt to build all models in the example/ directory
 # as tables. These settings can be overridden in the individual model files
 # using the `{{ config(...) }}` macro.
+    # Config indicated by + and applies to all files under models/example/
+    # example:
+      # +materialized: view
+
+# TODO configure project
 models:
   harmony:
-    # # Config indicated by + and applies to all files under models/example/
-    # example:
-    #   +materialized: view
+    core:
```

Comment:
Lets remove this.. I think it automatically crawls the folders inside "source-paths" for the ymls and the yml's has the same pattern. Was something not working that you needed to add this?

If we do need this.. materialized should be defaulted to incremental as that should be our default mode..

Reply:
I thought it was necessary 😅. Will remove